File: Day 76/Classwork/Cw1.py
#Number of People in the Bus
def number(bus_stops):
    total = 0
    for on, off in bus_stops:
        total += on
        total -= off
    return total

#Odd or Even?
def odd_or_even(arr):
    return "even" if sum(arr) % 2 == 0 else "odd"

#The highest profit wins!
def min_max(lst):
    return [min(lst), max(lst)]

File: Day 83/Classwork/Cw1.js
const numbers = [5, 8, 12, 7, 10, 15, 4];
let sum = 0;

for (let i = 0; i < numbers.length; i++) {
    sum += numbers[i];
    if (sum > 50) {
    break;
}
}

console.log("ჯამი:", sum);